cmd/kbagent/main.go

```diff
@@ -41,7 +41,8 @@ import (
 )
 
 const (
-	defaultPort           = 3501
+	defaultHTTPPort       = 3501
+	defaultStreamingPort  = 3502
```

Comment:
It will make the data streaming protocol more constrained and complicated if sharing a same port.

Reply:
it won't be constrained because http is just a transport layer. but the advantage is that it only uses one port, it's useful when the cluster is deployed under the host network. I'm fine with both, just raise an option for consideration.